tests/e2e/support/objects/runtime/session.ts

```diff
@@ -29,16 +29,10 @@ export class Session {
     await this.#page.locator('#kc-login').click()
   }
 
-  async login({
-    user,
-    tokenType = null
-  }: {
-    user: User
-    tokenType?: TokenProviderType
-  }): Promise<void> {
+  async login({ user }: { user: User; tokenType?: TokenProviderType }): Promise<void> {
```

Comment:
`tokenType` is not used
```suggestion
  async login({ user }: { user: User }): Promise<void> {
```